src/main.c

```diff
@@ -86,6 +88,8 @@ static int process_imu(const struct device *dev) {
     struct sensor_value temperature;
     struct sensor_value accel[3];
     struct sensor_value gyro[3];
+    uint64_t currentTime;
```

Comment:
Please use `snake_case` for variables for consistency.

Reply:
Also, it would be good to express the units this is in with a suffix (e.g. `uptime_ms`).

Reply:
And finally there is no reason to define this here and then assign a value later, you can just create a const value and immediately assign it a value.